package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_dum_ways_to_die",
+  "version": "1.0.0",
+  "description": "This is your first repository\r This is my first repository This is a project in which I will show some dumb ways to Dieeeee!",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
```

Comment:
Consider adding more specific keywords related to your project in the "keywords" array. This can help others find your project more easily. [medium]